src/model/model.js

```diff
@@ -695,14 +696,19 @@ export default class Model {
 			const callbackReturnValue = this._pendingChanges[ 0 ].callback( this._currentWriter );
 			ret.push( callbackReturnValue );
 
-			// Fire internal `_change` event.
+			// Collect an information whether the model document has changed during from the last pending change.
+			hasModelDocumentChanged = hasModelDocumentChanged || this.document._hasDocumentChanged();
+
+			// Fire '_change' event before resetting differ.
 			this.fire( '_change', this._currentWriter );
```

Comment:
It should be marked as deprecated.

Reply:
This event is marked as deprecated on top of that file.